fundamentals/streaming/SynchronousWithNewtonsoftJson/SynchronousWithNewtonsoftJson.csproj

```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>net7.0</TargetFramework>
+    <AssemblyName>ASPNetCoreStreamingExample.SynchronousWithNewtonsoftJson</AssemblyName>
+    <RootNamespace>ASPNetCoreStreamingExample.SynchronousWithNewtonsoftJson</RootNamespace>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.Mvc.NewtonsoftJson" Version="6.0.9" />
```

Comment:
```suggestion
    <PackageReference Include="Microsoft.AspNetCore.Mvc.NewtonsoftJson" Version="7.0.0-rc.1.22427.2" />
```

Reply:
This is fair, except that it's not going to be in RC forever. Is there a process whereby these samples get updated to refer to release versions when they become available?

Reply:
I don't know if there's a formal process, but dependencies should not be mixed across major versions. E.g. 6.0 shouldn't be used with net7, always use the 7.0 version.

Reply:
Mm, that's reasonable. I just don't like locking in the prerelease tag. But, if it's omitted, it won't automatically use a prerelease version, will it? 🤔

Reply:
We have to manually update to RTM.

Reply:
Does that mean that I can set it to the prerelease version today, and in the future it won't get forgotten when there's a full release? :-)

Reply:
> Does that mean that I can set it to the prerelease version today, and in the future it won't get forgotten when there's a full release? :-)

Yes, we'll take care of it.